elements/pfe-band/test/pfe-band_test.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
Is this file being used? It doesn't look like it.

Reply:
Yes, this is the main test file.  I deleted the js file which wasn't working anyway.